internal/model/board.go

```diff
@@ -259,38 +293,143 @@ func (b *Board) MoveCurrentPlayerTo(line, row int) error {
 	return nil
 }
 
-// CurrentPlayer returns the current player.
-func (b Board) CurrentPlayer() *Player {
+func (b Board) GetCurrentPlayer() *Player {
```

Comment:
Some function start in Uppercase, other not. is there an explanation ?

Reply:
This is du to the Go language design : Upper-Cased functyionas are public, Lower-Cased are private.